tools/build-nuget.ps1

```diff
@@ -0,0 +1,73 @@
+# Function returns 4-part 'classic' version string from SemVer 2.0 string
+function GenVer4Part([String] $Version)
+{
+  if ($Version[0] -eq 'v')
+  {
+    # If tag contains 'v' then strip it
+    $Version = $Version.substring(1)
+  }
+  # Converts from 1.2.3-build4 to 1.2.3.4
+  $Version = $Version -replace "-build", "."
+  $Version = $Version -replace "-", "."
+
+  $parts = $Version.Split('.');
+  # Add missing tuples
+  $i = $parts.Count
+  while($i -lt 4)
+  {
+    $parts += "0"
+    $i++
+  }
+
+  ,$parts
```

Comment:
What is this line for?

Reply:
It's a PowerShell syntax for returning an array back to the caller.